package.json

```diff
@@ -64,10 +64,11 @@
   },
   "dependencies": {
     "ethereumjs-util": "^6.1.0",
+    "ethers": "^4.0.29",
```

Comment:
Ethers should not be a dependency. Its a peer-dependency only. Otherwise JS typesystem cant handle typesafety :roll_eyes:

Reply:
Right. Now the unintended dependency is gone, and also version patch bumped.